src/collective/cover/utils.py

```diff
@@ -17,3 +18,29 @@ def assign_tile_ids(layout, override=True):
             children = elem.get('children')
             if children:
                 assign_tile_ids(children, override)
+
+
+def uuidToObject(uuid):
+    """Return a content object given an UUID.
+
+    :param uuid: UUID of the object
+    :type uuid: str
+    :returns: the content object or None, if the uuid can't be found.
+    """
+    # Use local uuidToCatalogBrain without the inactive content filter
+    brain = uuidToCatalogBrain(uuid)
+    return brain.getObject() if brain else None
+
+
+def uuidToCatalogBrain(uuid):
```

Comment:
@fredvd should we rename this one? ``uuidToCatalogBrainUnfiltered`` of something like that?